src/Types/index.ts

```diff
@@ -158,12 +158,30 @@ export type AuthLoginTypes = {
 export type TalentTypes = {
   id: string;
   key?: string;
+  firstName: string;
+  lastName: string;
   nama: string;
+  tanggalLahir: string;
+  domisili: string;
+  phone: string;
+  email: string;
   divisi: string;
   posisi: string;
+  kontrak: number;
   salary: number;
-  status: string;
   skills: string[];
+  bahasa: string[];
+  github: string;
+  linkedin: string;
+  websitePortfolio: string;
+  status: "idle" | "on going";
+  isAvailableWFO: boolean;
+  cv: {
+    file: {
+      uid: string;
+    };
+    fileList: FileUploadTypes[];
+  };
 };
```

Comment:
kalau dari BE ga provide, kita bikin standard in bahasanya, kalau inggris, inggris terus, kalau indonesia indonesia semua